src/components/transactions/TransactionList.vue

```diff
@@ -4,9 +4,9 @@
       <div :key="group[0].title">
         <h3>{{ group[0].title }}</h3>
         <TransactionItem
-          v-for="tx in group"
-          :key="tx.hash"
-          :transaction="tx.tx"
+          v-for="(item, index) in group"
+          :key="`${item.tx.hash}-${index}`"
```

Comment:
never use index. we can add message index in the API as an improvement

Reply:
> never use index. we can add message index in the API as an improvement

How would you build that index in api?